src/main/com/intellij/lang/jsgraphql/ide/documentation/GraphQLDocumentationProvider.java

```diff
@@ -58,10 +50,7 @@ public String getQuickNavigateInfo(PsiElement element, PsiElement originalElemen
 
     @Override
     public String generateDoc(PsiElement element, @Nullable PsiElement originalElement) {
-        if (isDocumentationSupported(element)) {
-            return createQuickNavigateDocumentation(element, true);
-        }
-        return null;
+        return createQuickNavigateDocumentation(element, true);
```

Comment:
Is the call to "isDocumentationSupported" no longer needed here?

Reply:
I think it's no longer needed, because `createQuickNavigationDocumentation` does that check already

Reply:
Ah, yeah, makes sense.